packages/framework-core/src/booster-command-dispatcher.ts

```diff
@@ -30,13 +31,13 @@ export class BoosterCommandDispatcher {
 
     const commandClass = commandMetadata.class
     this.logger.debug('Found the following command:', commandClass.name)
-    const command = new commandClass()
+    const command = commandClass as CommandClassInterface
     Object.assign(command, commandEnvelope.value)
     // TODO: Here we could call "command.validate()" so that the user can prevalidate
     // the command inputted by the user.
     const register = new Register(commandEnvelope.requestID, commandEnvelope.currentUser)
     this.logger.debug('Calling "handle" method on command: ', command)
-    await command.handle(register)
+    await command.handle(commandEnvelope.value, register)
```

Comment:
This won't work as expected, as we are not passing an instance of the command, but a plain object with its attributes, so if the user has some methods defined in its command class, it will fail. We should create the instance as we were doing, and pass it here

Reply:
Fixed https://github.com/boostercloud/booster/pull/333/commits/0705b5cf55a9a88872dbaebfc866e4f149737501 , thanks!